core/src/main/java/org/ldaptive/SearchOperation.java

```diff
@@ -827,10 +827,10 @@ public static SearchOperation copy(final SearchOperation operation)
     op.setExceptionHandler(operation.getExceptionHandler());
     op.setThrowCondition(operation.getThrowCondition());
     op.setUnsolicitedNotificationHandlers(operation.getUnsolicitedNotificationHandlers());
+    op.setConnectionFactory(operation.getConnectionFactory());
```

Comment:
Was this just for consistency with other copy methods? Does the order matter?

Reply:
Just for consistency, grouping the parent class methods together. Order doesn't matter.

Reply:
👍